app/src/main/java/com/keylesspalace/tusky/components/timeline/TimelineFragment.kt

```diff
@@ -215,6 +215,9 @@ class TimelineFragment :
                         if (loadState.append is LoadState.NotLoading && loadState.source.refresh is LoadState.NotLoading) {
                             binding.statusView.show()
                             binding.statusView.setup(R.drawable.elephant_friend_empty, R.string.message_empty, null)
+                            if (kind == TimelineViewModel.Kind.HOME) {
```

Comment:
Not sure if this is the proper location/technique to answer "timeline is empy"?

Reply:
I think it is.